magclassic/templates/regextra.html

```diff
@@ -1,12 +1,16 @@
 
 <script>
     if (window.BADGE_TYPES) {
-        window.BADGE_TYPES[0].description = 'Allows access to the festival for its duration.';
+        window.BADGE_TYPES.options[0].description = 'Allows access to the festival for its duration (Sept 11-13).';
     }
 
     $(function() {
         $('#tshirt-size-guide').insertAfter($.field('shirt'));
```

Comment:
This is an unrelated fix; because many of our forms don't have group fields, we only do the move if it exists and otherwise remove the explanation.